app/actions/websocket/preferences.ts

```diff
@@ -16,12 +20,14 @@ export async function handlePreferenceChangedEvent(serverUrl: string, msg: WebSo
     try {
         const preference = JSON.parse(msg.data.preference) as PreferenceType;
         handleSavePostAdded(serverUrl, [preference]);
+
         if (operator) {
             operator.handlePreferences({
                 prepareRecordsOnly: false,
                 preferences: [preference],
             });
         }
+        updateChannelDisplayName(serverUrl, msg.broadcast.user_id);
```

Comment:
What? No no, so here you need to identify if the preference is in fact the one for the display name format and the update EVERY DM/GM didplay name so that it follows the format